userguide/account.rst

```diff
@@ -168,46 +166,33 @@ operating system, as shown in
    Managing User Accounts
 
 
-Each account entry indicates the user ID, username, primary group ID,
-home directory, default shell, full name, whether it is a
-built-in user that came with the %brand% installation, the email
-address, whether logins are disabled, whether the user
-account is locked, whether the user is allowed to use
-:command:`sudo`, and if the user connects from a Windows
-8 or higher system. To reorder the list, click the desired column
-name. An arrow indicates which column controls the view sort order.
-Click the arrow to reverse the sort order.
+Each account entry indicates the username, user ID, primary group ID,
+home directory, default shell, and whether it is a built-in user that
+came with the %brand% installation. To reorder the list, click the
```

Comment:
Rearrange "To reorder the list" as "Clicking a column name will resort the list by that value."